pyproject.toml

```diff
@@ -43,6 +43,16 @@ jupyter = [
     'ipytree>=0.2.2',
     'ipywidgets>=8.0.0',
 ]
+docs = [
+    'sphinx',
+    'sphinx_design',
+    'sphinx-issues',
+    'sphinx-copybutton',
+    'sphinx-rtd-theme',
+    'pydata-sphinx-theme',
+    'numpydoc',
+    'numcodecs[msgpack]',
```

Comment:
Since `msgpack` is a `numcodecs` requirement, specify it via `numcodecs`' extra requirement